packages/firestore/src/platform_browser/browser_platform.ts

```diff
@@ -72,4 +74,23 @@ export class BrowserPlatform implements Platform {
   btoa(raw: string): string {
     return btoa(raw);
   }
+
+  randomBytes(nBytes: number): Uint8Array {
+    debugAssert(nBytes >= 0, 'Number of random bytes must not be negative');
+
+    // Polyfill for IE and WebWorker
```

Comment:
Can you improve this comment to explain how it affects the reader? Is the polyfill being done by the code block that follows? Or is it instructing the reader to perform some polyfilling? Consider re-writing this comment as a sentence that would be either clearly informative or clearly actionable by the reader.